### ISSUE TITLE
Feature: "alldifferent" constrant

### ISSUE THREAD

**Issue by kakukkokkal** [rust-or/good_lp#52]
Is it possible to add support for the "alldifferent" constraint where the solver supports it?

**Comment by lovasoa** [comment]
Yes, that would be nice. Which solver do you have in mind? A pull request would be welcome!

**Comment by lmmx** [comment]
[Coin CBC](https://github.com/coin-or/Cbc/blob/b5a21d4ae045513f0a8b3de21d6df18917ed1ddb/src/CbcBranchAllDifferent.cpp#L27) supports this, as does Pumpkin (#95)
